### PR TITLE
Translate to Portuguese the Solidity course - Section 1 - Lesson 3

### PR DESCRIPTION
**Pull request by igorln**

### CODE REVIEW

File: Solidity_And_Smart_Contracts/pt/Section_1/Lesson_3_Compile_Contract_Locally.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+Você já fez isso. Você já escreveu um contrato inteligente. Você é um campeão!
+
+Agora precisamos realmente\
+1\. Compilar.\
+2\. Fazer o deploy em nossa blockchain local.\
+3\. Quando estiver lá, esse console.log será executado :).
+
+Precisamos fazer isso porque no mundo real, os contratos inteligentes vivem na blockchain. E queremos que nosso site e contrato inteligente sejam usados por pessoas reais para que eles possam 👋 para nós ou fazer o que você quiser que elas façam!
+
+Então, mesmo quando estamos trabalhando localmente, queremos imitar esse ambiente. Tecnicamente, poderíamos apenas compilar e executar o código Solidity, mas o que torna o Solidity mágico é como ele pode interagir com as blockchain e carteiras Ethereum (que veremos mais na próxima lição). Então, é melhor acabar com isso agora.
```
Comment (daniserafs) [inline Solidity_And_Smart_Contracts/pt/Section_1/Lesson_3_Compile_Contract_Locally.md:13]: typo:
```suggestion
Então, mesmo quando estamos trabalhando localmente, queremos imitar esse ambiente. Tecnicamente, poderíamos apenas compilar e executar o código Solidity, mas o que torna o Solidity mágico é como ele pode interagir com a blockchain e carteiras Ethereum (que veremos mais na próxima lição). Então, é melhor acabar com isso agora.
```

File: Solidity_And_Smart_Contracts/pt/Section_1/Lesson_3_Compile_Contract_Locally.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+await waveContract.deployed();
+```
+
+Vamos esperar até que nosso contrato seja oficialmente deployado em nossa rede blockchain local! Nosso `constructor` é executado quando realmente implantamos.
+
+```javascript
+console.log("Contrato deployado para:", waveContract.address);
+```
+
+Finalmente, uma vez deployado, o `waveContract.address` basicamente nos dará o endereço do contrato implantado. Este endereço é como podemos realmente encontrar nosso contrato no blockchain. Existem milhões de contratos na rede blockchain de verdade. Assim, este endereço nos dá acesso fácil ao contrato com o qual estamos interessados em trabalhar! Isso será mais importante um pouco mais tarde, quando deployarmos em uma rede Ethereum real.
```
Comment (daniserafs) [inline Solidity_And_Smart_Contracts/pt/Section_1/Lesson_3_Compile_Contract_Locally.md:82]: I believe it's a typo:
```suggestion
Finalmente, uma vez deployado, o `waveContract.address` basicamente nos dará o endereço do contrato implantado. Este endereço é como podemos realmente encontrar nosso contrato na blockchain. Existem milhões de contratos na rede blockchain de verdade. Assim, este endereço nos dá acesso fácil ao contrato com o qual estamos interessados em trabalhar! Isso será mais importante um pouco mais tarde, quando deployarmos em uma rede Ethereum real.
```

**Review by daniserafs — COMMENTED** [review]
pretty good!

**Comment by igorln** [comment]
@daniserafs fixed, thanks for the help! :)

**Comment by adilanchian** [comment]
Yo @igorln @daniserafs ! Sorry for the late reply here :)

DM me on Discord! would love to chat more about translations `alec#8853`

**Comment by helloitsm3** [comment]
Closing as we do not have any way to verify the contents at the moment. Do re-open if you feel that this needs another review.

**Pull request closed without merging**